main.py

```diff
@@ -1,15 +1,20 @@
 from sketchy import Sketchy
 from ball import Ball
+from vector import Vector
 
 class MyDrawing(Sketchy):
 
     def setup(self):
-        return
+        width, height = [600, 400]
```

Comment:
Don't worry about this for now, I will show you how to setup unit tests for the class later.